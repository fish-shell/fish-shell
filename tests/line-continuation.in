ech\
o echo

buil\
tin echo builtin echo

true; an\
d echo true

\i\
f\
 true
	echo if true
\
\145n\
d\
;

'if'\
 true
	echo if true
"\
en\
d\
";
