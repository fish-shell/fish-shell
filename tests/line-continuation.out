echo
builtin echo
true
if true
if true
